test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ParallelExecutionTests.cs

```diff
@@ -52,6 +54,70 @@ public static async Task ParallelQueueTrigger([QueueTrigger(TestQueueName)] int
             }
         }
 
+        [Theory]
+        // One is special case (the old behaviour)
+        [InlineData(1)]
+        // Odd and even values
+        [InlineData(2)]
+        [InlineData(3)]
+        public async Task MaxDegreeOfParallelism_Queues_nonDynamicSku(int batchSize)
```

Comment:
nit: fix casing

Reply:
Also, rather than copy this test, can't you just add a new param to the existing test for sku? E.g. perhaps just a bool isDynamic

Reply:
done